packages/volto/cypress/tests/core/content/a11y.js

```diff
@@ -0,0 +1,100 @@
+describe('Accessibility Tests Content Types', () => {
+  beforeEach(() => {
+    cy.autologin();
+    cy.visit('/');
+    cy.injectAxe(); // make sure axe is available on the page
+  });
+
+  it('Event has no a11y violations', () => {
+    cy.get('#toolbar-add').click();
+    cy.get('#toolbar-add-event').click();
+    cy.get('.documentFirstHeading').type('Test Event Content Type');
+
+    cy.get('#toolbar-save').click();
+
+    cy.wait(1000);
```

Comment:
this looks fine however we will look if we can get rid of these wait timeouts to make it faster.

Let's also rename the test names to "[Content type] tested for a11y axe violations"